scripts/generate.ts

```diff
@@ -8,7 +8,7 @@ import { sys } from "typescript";
 
 const ROOT = path.resolve(__dirname, "..");
 const OUTPUT_PATH = "dist";
-const VULNERABILITIES_FOLDER = "vulnerabilities";
+const VULNERABILITIES_FOLDER = "risks";
```

Comment:
nit: rename to `RISKS_FOLDER`, as well as some variable names like `const vulns`